public/lang/babele/de/investigator.pathOfCthulhuAbilities.json

```diff
@@ -210,22 +210,22 @@
         },
         "Reassurance": {
             "name": "Beruhigen",
-            "category": "Sozial"
+            "category": "Zwischenmenschlich"
         },
         "Riding": {
             "name": "Reiten",
             "category": "Allgemein"
         },
         "Sanity": {
-            "name": "Geistige Gesundheit",
+            "name": "Verstand",
             "category": "Allgemein"
         },
         "Scuffling": {
-            "name": "Handgemenge",
```

Comment:
And once Again: Lets stay with terms already established. Also Raufen is basically unarmed fighting while Scuffling also utilizies weapons so Handgemenge is IMHO a bit more accurate.